backend/lib/services/shoots.js

```diff
@@ -119,15 +119,15 @@ exports.replacePurpose = async function ({ user, namespace, name, body }) {
   return client['core.gardener.cloud'].shoots.mergePatch(namespace, name, payload)
 }
 
-exports.patchBindingResource = async function ({ user, namespace, name, body }) {
+exports.replaceSeedName = async function ({ user, namespace, name, body }) {
   const client = user.client
   const seedName = body.seedName
   const payload = {
     spec: {
       seedName
     }
   }
-  return client['core.gardener.cloud'].shoots.patchBindingResource(namespace, name, payload)
+  return client['core.gardener.cloud'].shoots.mergePatch(namespace, [name, 'binding'], payload)
```

Comment:
In the other cases we use `jsonPatch` if the parent property always exists
```js
  const patchOperations = [{
    op: 'replace',
    path: '/spec/seedName',
    value: seedName
  }]
  return client['core.gardener.cloud'].shoots.jsonPatch(namespace, name, patchOperations)
```

Reply:
ok, changed it to `jsonPatch`